common/lib/capa/capa/templates/textline.html

```diff
@@ -57,7 +57,7 @@
 % endif
 
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-describedby="answer_input_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
"answer_input_${id}" does not appear elsewhere in this template. Is it a legitimate value to be using?